src/styles/themes/selectBox.ts

```diff
@@ -0,0 +1,71 @@
+import type { ThemeOption } from '@types'
+
+type SelectBoxKeys = keyof typeof selectBox
+type SelectBoxValues = typeof selectBox[SelectBoxKeys]
+
+export type SelectBox = ThemeOption<SelectBoxKeys, SelectBoxValues>
+
+export const selectBox = {
+  medium: {
+    default: `
+      div:nth-child(1){ //triggerWrapper
+        height: 40px;
+        padding: 12px 10px;
+        border-radius: 6px;
+        font-size: 14px; font-weight: 500; line-height: 20px;
+      }
+      div:nth-child(2){ //StyledOptionsWrapper
+        height: 40px;
+        padding: 12px 10px;
+        border-radius: 6px;
+        font-size: 14px; font-weight: 500; line-height: 20px;
+        top: 48px;
+    }`,
+    isEmpty: `
+        div:nth-child(1){ //triggerWrapper
+          height: 40px;
+          padding: 12px 10px;
+          border-radius: 6px;
+          font-size: 14px; font-weight: 500; line-height: 20px;
+          color: #929399;
```

Comment:
```suggestion
          font-size: 14px; font-weight: 500; line-height: 20px;
          color: #929399;
```
요기도 아래와 동일하게 theme객체 활용하면 좋을 것 같아요!